packages/stopwatch/src/tsconfig.json

```
@@ -1,6 +1,7 @@
 {
 	"extends": "../../ts-config/extra-strict-without-decorators.json",
 	"compilerOptions": {
+		"lib": ["dom", "dom.iterable", "esnext"],
```

```suggestion
		"lib": ["dom", "esnext"],
```

Doesn't `dom` also include `dom.iterable`?